container-host-files/opt/scf/bin/klog.sh

```diff
@@ -5,192 +5,82 @@ set -e
 KLOG=${HOME}/klog
 
 if [ "$1" == "-h" ]; then
-    cat <<EOF
+	cat <<EOF
 usage: $0 [-f] [-v] [INSTANCE_ID]
 
   -f  forces fetching of all logs even if a cache already exists
 
   INSTANCE_ID defaults to "scf"
 EOF
-    exit
+	exit
 fi
 
 FORCE=0
 if [ "$1" == "-f" ]; then
-    shift
-    FORCE=1
+	shift
+	FORCE=1
 fi
 
 NS=${1-scf}
 DONE="${KLOG}/${NS}/done"
 
-if [ "${FORCE}" == "1" ] ; then
-    rm -f "${DONE}" 2> /dev/null
+if [ "${FORCE}" == "1" ]; then
+	rm -f "${DONE}" 2>/dev/null
 fi
 
 function get_phase() {
-  kubectl get pod "${POD}" --namespace "${NS}" --output=jsonpath='{.status.phase}'
+	kubectl get pod "${POD}" -n "${NS}" -o jsonpath='{.status.phase}'
```

Comment:
Please don't use short options in scripts; it's (typically) harder to see at a glance what they're doing if it's a tool that we haven't used manually in a while. (This probably doesn't apply to `kubectl`, but the standard style is long argument names in scripts)

Reply:
Sounds Legit. Reverted short options.